docs/index.html

```diff
@@ -55,14 +55,8 @@ <h3>Stackdriver Sandbox is an open source tool* that helps practitioners to lear
       <h2>Get started now</h2>
       <div class="steps">
         <div class="step step-1">
-          <label>Step 1</label>
-          <a class="console-btn" href="https://console.cloud.google.com/cloudshell/editor?cloudshell_git_repo=https://github.com/GoogleCloudPlatform/stackdriver-sandbox.git&cloudshell_git_branch=v0.1.14&cloudshell_working_dir=terraform&cloudshell_image=gcr.io/cloudshell-images/cloudshell" target="_blank" onclick="metrics.recordCustomTime('stackdriver-demo-open-in-cloud-shell')">Open in Google Cloud Shell</a>
-        </div>
-	<div class="step step-2">
-          <label>Step 2</label>
-          <p>In the Cloud Shell command prompt, type this command to install the Sandbox</p>
-          <code onclick="navigator.clipboard.writeText('./install.sh'); metrics.recordCustomTime('stackdriver-demo-copy-code-for-cloud-shell')">./install.sh</code>
-          <div class="line"></div>
+          <!--label>Step 1</label-->
+          <a class="console-btn" href="https://console.cloud.google.com/cloudshell/editor?cloudshell_git_repo=https://github.com/GoogleCloudPlatform/stackdriver-sandbox.git&cloudshell_git_branch=v0.1.14&cloudshell_working_dir=terraform&cloudshell_image=gcr.io/stackdriver-sandbox-230822/cloudshell-image" target="_blank" onclick="metrics.recordCustomTime('stackdriver-demo-open-in-cloud-shell')">Open in Google Cloud Shell</a>
```

Comment:
I noticed the image here has a different path than the one used in the README

Reply:
This is a good note! I updated this file before updating the README. Now they're both up-to-date.